R/tidy_FIA.R

```diff
@@ -67,7 +67,7 @@ tidy_fia <- function(states = NULL, aoi = NULL, postgis = TRUE,
     con <- DBI::dbConnect(
       RPostgres::Postgres(),
       dbname = "fiadb",
-      host = "fiadb.csrjp3emmira.us-east-1.rds.amazonaws.com",
+      host = ncxfiadb_host,
       port = 5432,
       user = "tidyfia",
```

Comment:
Should we use the `readonly` user here?

Reply:
Branson made a 'readonly' ROLE, then granted those permissions to the 'tidyfia' user so I think we are good to go!

Reply:
![6laow5](https://user-images.githubusercontent.com/18391567/176550035-85117f67-9757-498f-9632-386ea8f6b197.jpg)

Reply:
@BKDaugherty made a 'readonly' ROLE, then granted those permissions to the 'tidyfia' user so I think we are good to go!